patches/react-autosize-textarea+7.1.0.patch

```diff
@@ -6,8 +6,8 @@ index ad6ff3f..e0a6d51 100644
          lineHeight: number | null;
```

Comment:
This change seems to cause a lot of CI failures. I think this file cannot be modified because it is a patch for an external npm library.